index.js

```diff
@@ -0,0 +1,24 @@
+"use strict";
+
+module.exports = {
+  extends: ["plugin:@typescript-eslint/recommended"],
+  rules: {
+    "@typescript-eslint/explicit-function-return-type": 0,
+    "@typescript-eslint/no-unused-vars": 0,
+    "@typescript-eslint/explicit-member-accessibility": 0,
+    "@typescript-eslint/no-explicit-any": 0,
+    "@typescript-eslint/camelcase": 0,
+    "@typescript-eslint/ban-ts-ignore": 0,
+    "@typescript-eslint/no-non-null-assertion": 0,
+    "@typescript-eslint/explicit-module-boundary-types": 0,
+    "@typescript-eslint/ban-ts-comment": 0,
+    // Core `no-shadow` rule reports incorrect errors for enum declarations
+    "no-shadow": "off",
+    "@typescript-eslint/no-shadow": ["error"],
+    "no-use-before-define": "off",
+    "@typescript-eslint/no-use-before-define": [
+      "error",
+      {functions: false, classes: true}
+    ]
+  },
```

Comment:
Should we add `"no-undef": 0` 🤔 ? We have this note👇  in every eslint config but did not do anything to turn off this rule in our packages. What do you say? @edizcelik @yasincaliskan 

```js
    // tsc already catches these kind of errors
    "no-undef": 0
```

Reply:
Yep let's override that too